sklearn/model_selection/_search.py

```diff
@@ -1411,6 +1411,21 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> from sklearn.model_selection import RandomizedSearchCV
+    >>> rng = np.random.RandomState(0)
+    >>> iris = load_iris()
+    >>> logistic = LogisticRegression(max_iter=200)
+    >>> hyperparameters = dict(C=rng.uniform(low=0, high=4, size=5),
```

Comment:
I am wondering about the usefulness of this. 

I would expect to either perform a random search on a regular grid or an exhaustive search on a random grid. Randomizing both sounds a bit too much. Or is it a thing that is done @glemaitre ?

Reply:
Uhm true. I thought to pass the distribution at first but we should use `scipy.stats.uniform` instead, isn't it?